tex/chProofs.tex

```diff
@@ -2116,8 +2209,8 @@ \subsection{Proofs by induction}\label{ssec:ind}
 of the art of search tools for formal libraries is far from being as
 advanced as what exists for instance for the world wide web.
 
-In order to help the users finding their needle in the haystack,
-the \mcbMC{} library follows uniform naming policies and the Ssreflect
+In order to help the users find their needle in the haystack,
```

Comment:
I think this is a legitimate modification of the idiom -- this particular haystack has lots of interesting needles.